docs/src/references.bib

```diff
@@ -5,6 +5,13 @@
 % --- A
 %
 %
+@article{dodd2024learning,
```

Comment:
```suggestion
@article{DoddSharrockNemeth:2024,
```

This would be the unified naming scheme we use. Could we use that here as well?
Also the literature is ordered by first-authors-lastname.

Reply:
ah sure :). I just assumed you already have done it so I just hit the button. No problem.

Reply:
yeah, sorry for not being precise.
If I could've, I would have – but I can not push to this branch, so all I can change is files that were already changed – and for those even a bit limited with the frontend here.

Reply:
ok now it is correct.